tests/device_tests/ethereum/test_definitions_bad.py

```
@@ -32,108 +36,201 @@ def fails(session: Session, network: bytes, match: str) -> None:
+            **get_clear_signing_sign_tx_params(),
+            definitions=messages.EthereumDefinitions(
+                encoded_erc7730_display_format=erc7730_display_format,
+            ),
+        )
+
+
+def _make_erc7730_payload(
+    timestamp: int = 0xFFFF_FFFF,
+    message: (
+        messages.EthereumERC7730DisplayFormatInfo | bytes
+    ) = make_eth_erc7730_display_format(),
```

Ruff (0.15.10)

[warning] Do not perform function call `make_eth_erc7730_display_format` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)